src/NetworkRequests/APIRequests.js

```diff
@@ -17,6 +17,28 @@ export const getAllMovies = async () => {
 
 export const getSingleMovieDetails = async (movieID) => {
   const response = await fetch(`https://rancid-tomatillos.herokuapp.com/api/v2/movies/${movieID}`)
-  const data = await response.json();
+  const data = await response.json()
   return data;
 }
+
+export const getAllUserRatings = async (userId) => {
+  const response = await fetch(`https://rancid-tomatillos.herokuapp.com/api/v2/users/${userId}/ratings`)
+  const ratings = await response.json()
+  return ratings
+}
+
+export const addMovieRating = async (userId, id, rating) => {
+  const response = await fetch(`https://rancid-tomatillos.herokuapp.com/api/v2/users/${userId}/ratings`, {
+      "method": "POST",
+      "headers": {
+          "content-type": "application/json"
+      },
+      "body": JSON.stringify({
+        "movie_id": id,
+        "rating": rating
+      })
+    }
+  )
+  const message = await response.json()
+  return message
+}
```

Comment:
Great start to this file! Add try/catch blocks around your async await calls and keep adding any calls you make to the api in this file.